src/main/java/com/owncloud/android/lib/resources/files/SearchRemoteOperation.java

```diff
@@ -43,12 +44,12 @@
 /**
  * Remote operation performing the search in the Nextcloud server.
  */
-public class SearchRemoteOperation extends RemoteOperation {
+public class SearchRemoteOperation extends RemoteOperation<ArrayList<RemoteFile>> {
```

Comment:
Any reason to go for `ArrayList` instead if its interface `List`?

Reply:
Good catch